ci-operator/config/openshift/console/openshift-console-master.yaml

```diff
@@ -9,6 +9,9 @@ images:
 - dockerfile_path: Dockerfile
   from: base
   to: console
+- dockerfile_path: Dockerfile.plugins.demo
+  from: base
+  to: plugin-pull-spec
```

Comment:
I'd call this something like `console-demo-plugin`

Reply:
@spadgett ok thanks, done